spec/models/form/question_spec.rb

```diff
@@ -107,7 +107,7 @@
     it "has an update answer link text helper" do
       expect(subject.update_answer_link_name(case_log)).to eq("Answer")
       case_log[question_id] = 5
-      expect(subject.update_answer_link_name(case_log)).to eq("Change")
+      expect(subject.update_answer_link_name(case_log)).to include("Change")
```

Comment:
Might be better to test for the visually hidden link as well? TDD type approach to this change?

Reply:
I've updated the test accordingly 😄